### DISCUSSION TITLE
problems in starting raisim unity

### DISCUSSION THREAD

**Discussion by joon725** [raisimTech/raisimLib#440] [General]
Before starting my questions, thank you for providing such a great software.

I was trying to use raisim for academic purpose.
However, i met some problems while doing it.

1. I installed CMAKE and did exactly the same way following the video in youtube.
 : Configuring and Generating worked but when i pressed Open Project, nothing happened.(Visual Studio didn't appear)

2. So i tried to open some example files directly through VS, then building of example files worked. But when i tried to show them in raisim unity, i got this error ; System.Exception : Cannot find mesh from resource directories = meshes/trunk.dae
this error was occured when i tried to open aliengo.cpp
also, when i tried to start anymals_stress_test.cpp, building failed. 
This error followed :  error : FindFirstFileExA(../../../../../바탕 화면/raisimlib/raisim/win32/include/ode): ~~~

I did everything following the youtube video but i don't know why these errors keep pop out.
Can you help me how to solve it?

**Comment by TheotimeLH** [comment]
I have the same error, and we tried to find the reason.

Empirically RaisimUnity isn't compatible with Ubuntu 22.04. Can you confirm that you are also using Ubuntu 22.04 ?

I haven't found a solution, how about you ? 
I will try to use RaisimUnreal.
